rtmidi/midiutil.py

```diff
@@ -14,7 +14,7 @@
 
 __all__ = [
     'get_api_from_environment',
-    'list_avaiilable_ports',
+    'list_available_ports',
```

Comment:
Argh, they've changed the comment interface and I can't figure out how to select multiple lines!